integration/app/app.server.module.ts

```diff
@@ -7,13 +7,7 @@ import { AppComponent } from './app.component';
 import { AppModule } from './app.module';
 
 @NgModule({
-  imports: [
-    AppModule,
-    ServerModule,
-    NoopAnimationsModule,
-    ServerTransferStateModule,
-    ModuleMapLoaderModule
-  ],
+  imports: [AppModule, ServerModule, NoopAnimationsModule, ServerTransferStateModule, ModuleMapLoaderModule],
```

Comment:
I think that 120 characters is too long.
It's defined in .prettierrc.json

Reply:
I agree. I think that 90 or 100 would be better. I actually have a PR waiting with this change but I'm waiting for some of the other outstanding ones to get merged to avoid conflicts

Reply:
I agree. I think that 90 or 100 would be better. I actually have a PR waiting with this change but I'm waiting for some of the other outstanding ones to get merged to avoid conflicts

Reply:
I'll resolve the prettier length soon. 
TBH I really don't like the way that prettier uses line length as the primary target. It ends up uglifying so many nicely formated lines by squashing them together.